src/dataflow/src/decode/csv.rs

```diff
@@ -54,7 +54,7 @@ where
             let mut buffer = vec![0u8];
             let mut bounds = vec![0usize];
             let mut csv_reader = csv_core::ReaderBuilder::new().delimiter(delimiter).build();
-            let mut row_packer = repr::RowPacker::new();
+            let mut row_packer = Row::default();
```

Comment:
Could we call this `row` instead of `row_packer`?

Reply:
I'd like that too but I don't think it will work - e.g. [this line](https://github.com/MaterializeInc/materialize/pull/6276/files#diff-6397477f722c4b72ea18b0a776cedfd87d20e63cad7964d084ec795c01eff91bR129) would turn weird (two row variables?). ):

Reply:
Fwiw, this is pretty pervasive through the PR, converting `row_packer` types, but not otherwise reworking the surrounding code. The general intent/style is that if we re-use a thing, it should be `row_packer` and otherwise `row` makes sense.

Reply:
And in this case the `finish_and_reuse()` part at the end means we re-use it. Could certainly go through and try to come up with better names, I agree!

Reply:
That explanation makes a bunch of sense, thanks!